lib/src/empty_unmodifiable_set.dart

```diff
@@ -34,7 +34,7 @@ class EmptyUnmodifiableSet<E> extends IterableBase<E>
   E singleWhere(bool Function(E) test, {E Function()? orElse}) =>
       orElse != null ? orElse() : throw StateError('No element');
   @override
-  Iterable<T> whereType<T>() => Iterable.empty();
+  Iterable<T> whereType<T>() => const Iterable.empty();
```

Comment:
This is not semantics preserving. The original returned an `Iterable<T>.empty()`, this returns an `Iterable<Never>.empty()`.

It matters if someone does `.toList().add(aTValue)`.
(It probably doesn't, but it *could*.)

So add the `<T>` instead of the `const`.

Reply:
Done!